handsontable/src/editors/textEditor/textEditor.js

```diff
@@ -387,26 +394,13 @@ export class TextEditor extends BaseEditor {
    * @private
    */
   bindEvents() {
-    this.eventManager.addEventListener(this.TEXTAREA, 'cut', event => event.stopPropagation());
-    this.eventManager.addEventListener(this.TEXTAREA, 'paste', event => event.stopPropagation());
-
     if (isIOS()) {
       // on iOS after click "Done" the edit isn't hidden by default, so we need to handle it manually.
       this.eventManager.addEventListener(this.TEXTAREA, 'focusout', () => this.finishEditing(false));
     }
 
     this.addHook('afterScrollHorizontally', () => this.refreshDimensions());
     this.addHook('afterScrollVertically', () => this.refreshDimensions());
-
-    this.addHook('afterColumnResize', () => {
-      this.refreshDimensions();
-      this.focus();
-    });
-
-    this.addHook('afterRowResize', () => {
-      this.refreshDimensions();
-      this.focus();
-    });
```

Comment:
I thing this ones need to be reverted. Otherwise there is a bug:
![Kapture 2023-05-10 at 09 20 20](https://github.com/handsontable/handsontable/assets/571316/b72fd4fd-48e4-4600-b78c-f1ba33716a7d)
As the tests pass after that regression can I ask to add some that would cover that feature?

Reply:
👍  I'm actually glad you found it 😆 because I was not sure if these callbacks were needed, but couldn't find a way to reproduce them myself (and as you said, the tests passed regardless)